src/slices-and-lifetimes/exercise.md

```diff
@@ -47,7 +47,7 @@ What remains for you is to implement the `parse_field` function.
 
 
 {{#include exercise.rs:parse_field }}
-    // 1. Read and unpack the tag.
+    // 1. Read and unpack the tag, which is a varint.
```

Comment:
Students noted confusion over how to get the tag, and this detail _is_ present in the docs, but not obvious, so this seems like a fair hint